app/controllers/concerns/metadata_template_actions.rb

```diff
@@ -112,6 +127,12 @@ def metadata_templates_path
 
   private
 
+  def metadata_templates
```

Comment:
Can we update this method name to be something like `metadata_templates_ancestral` and then change the `load_metadata_templates` method to `load_namespace_metadata_templates`

Reply:
Updated in ab4de3e